src/index.js

```diff
@@ -130,9 +132,10 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
           events = [events];
         }
         events.forEach(eventName => {
-          const handlerOptions = !this.props.preventDefault && ['touchstart', 'touchmove'].indexOf(eventName) !== -1
-            ? { passive: true }
-            : null;
+          const handlerOptions =
```

Comment:
there is nothing pretty about this tertiary. Instead of making it "an object or null", let's do this instead:
```
const handlerOptions = {
  passive: false
};

const touchEvents = [
  'touchstart',
  'touchmove'
];

if (!this.props.preventDefault && touchEvents .indexOf(eventName) > -1) {
  handlerOptions.passive = true;
}
```